src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -97,6 +97,9 @@ internal class RazorLanguageServerClient : ILanguageClient, ILanguageClientCusto
         public async Task<Connection> ActivateAsync(CancellationToken token)
         {
             var (clientStream, serverStream) = FullDuplexStream.CreatePair();
+
+            await EnsureCleanedUpServerAsync(token).ConfigureAwait(false);
```

Comment:
/cc @alexgav @jimmylewis @ToddGrun @dibarbet you guys may want to workaround these as well given the LSP platform won't be able to address this until 16.8-Preview2.